dts/bindings/i2c/wch,i2c.yaml

```diff
@@ -0,0 +1,16 @@
+# Copyright (c) 2025 Andrei-Edward Popa
+# SPDX-License-Identifier: Apache-2.0
+
+description: WCH I2C controller
+
+compatible: "wch,i2c"
+
+include: [i2c-controller.yaml, pinctrl-device.yaml]
+
+properties:
+  reg:
+    required: true
+  clocks:
```

Comment:
nordicjm would say you need newlines between the properties.

Reply:
I put newlines